sql/core/src/test/resources/tpcds-query-results/v1_4/q23a.sql.out

```diff
@@ -3,4 +3,4 @@
 -- !query schema
 struct<sum(sales):decimal(28,2)>
 -- !query output
-17030.91
+NULL
```

Comment:
I've checked that the result is the same when using StringType instead of CharType. It looks sf=1 is not enough to output valid rows, e.g., if sf=20, this result will be;
```
213230904.79
```